<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartHomeModisClass</class>
 <widget class="QMainWindow" name="SmartHomeModisClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartHomeModis</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_smarthome">
       <attribute name="title">
        <string>SmartHome</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="LED3_4">
          <property name="maximumSize">
           <size>
            <width>720</width>
            <height>480</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="smarthomemodis.qrc">:/Resources/smarthome-1.jpg</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_saunaroom">
       <attribute name="title">
        <string>SaunaRoom </string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="MAC">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>80</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>28</pointsize>
           </font>
          </property>
          <property name="text">
           <string>MAC</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QWidget" name="widget" native="true">
              <property name="maximumSize">
               <size>
                <width>400</width>
                <height>100</height>
               </size>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>LED0</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="label_2">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>LED1</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QLabel" name="label_3">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>LED2</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="3">
                <widget class="QLabel" name="label_4">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>LED3</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="LED0">
                 <property name="maximumSize">
                  <size>
                   <width>31</width>
                   <height>31</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="smarthomemodis.qrc">:/Resources/on_green.png</pixmap>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="LED1">
                 <property name="maximumSize">
                  <size>
                   <width>31</width>
                   <height>31</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="smarthomemodis.qrc">:/Resources/on_green.png</pixmap>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QLabel" name="LED2">
                 <property name="maximumSize">
                  <size>
                   <width>31</width>
                   <height>31</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="smarthomemodis.qrc">:/Resources/on_green.png</pixmap>
                 </property>
                </widget>
               </item>
               <item row="1" column="3">
                <widget class="QLabel" name="LED3">
                 <property name="maximumSize">
                  <size>
                   <width>31</width>
                   <height>31</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="smarthomemodis.qrc">:/Resources/on_green.png</pixmap>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap>:/Resources/ready_yellow.png</pixmap>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="FAN1_lable">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="smarthomemodis.qrc">:/Resources/fan.gif</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="lcdNumber">
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>100</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="FAN2_lable">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="smarthomemodis.qrc">:/Resources/fan.gif</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_setting">
       <attribute name="title">
        <string>setting</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QTextEdit" name="debugTv">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>300</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="smarthomemodis.qrc"/>
 </resources>
 <connections/>
</ui>
